py-polars/polars/internals/series/series.py

```diff
@@ -4250,6 +4612,18 @@ def set_sorted(self, reverse: bool = False) -> Series:
         This can lead to incorrect results if this `Series` is not sorted!!
         Use with care!
 
+        Examples
+        --------
+        >>> s = pl.Series("a", [1, 3, 2])
+        >>> s.sort().set_sorted()
```

Comment:
Not sure this is the right example here. I think it makes more sense to define a series that is inately sorted (e.g. `s = pl.Series("a", [1, 2, 3])` , then call `set_sorted()` on it. That is the intended usage. In your example, after calling `sort()`, the series will already have the `sorted` flag set to True.

Reply:
Great idea, I've changed this to set_sorted().max() for in both series.py and expr.py

Reply:
@stinodego - I've realised that lots of examples were lost when I merged with master recently. I can recover them from my local git but it might take a few days to get everything back in place

Reply:
That's unfortunate - git can be a bit tricky sometimes.

I'd advise to fix the two errors I pointed out (duplicate example sections), then we can merge this PR. Your other docs examples can be merged in a separate PR.

Reply:
Hi @stinodego I've recovered the other changes. I've included them here as they are all in a separate file (series/datetime.py) and so don't require you to re-review any files. Hope that's ok.

I got some strange CI issues earlier (one was just a timeout, the other a indentation problem that isn't apparent from inspecting the code. I'll @ when I get it sorted.

Reply:
@stinodego CI is passing now, should be good to go